src/components/SharedComponents/PhotoCount.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+import { View } from "components/styledComponents";
+import * as React from "react";
+import Svg, { Path, Text } from "react-native-svg";
+
+type Props = {
+  count: number,
+  size?: number,
+  shadow: boolean,
+};
+
+const PhotoCount = ( { count, size, shadow }: Props ): React.Node => {
+  if ( count <= 0 || count > 99 ) {
+    return null;
+  }
+
+  return (
+    <View
+      className={`drop-shadow-md w-[${String( size )}px] h-[${String( size )}px] ${String(
+        shadow && "shadow"
+      )}`}
+    >
+      <Svg
+        height={size}
+        width={size}
+        viewBox="0 0 24 24"
```

Comment:
If the viewBox is 24x24 and the icon is smaller it will be aligned at 0,0 (so top left) correct? If that is true we have to change the viewBox by size I guess.

Reply:
No need to viewBox, I tried and it works fine.
changing viewBox by size make ui issue

Reply:
I don't understand. The viewBox prop is still there. Is it required? Expo documents mention to best have it included for Android. Currently, it is still present in the component. Will there be a problem with diferent sizes?

Reply:
Yes, the viewBox is required.
There are no issues if the icon size is smaller or bigger than 24